src/NuGet.Services.EndToEnd/Support/Clients/GalleryClient.cs

```diff
@@ -22,6 +26,24 @@ public GalleryClient(TestSettings testSettings)
             _testSettings = testSettings;
         }
 
+        public async Task<IList<string>> AutocompletePackageIdsAsync(string id, bool includePrerelease, string semVerLevel, ITestOutputHelper logger)
+        {
+            var serviceEndpoint = $"{_testSettings.GalleryBaseUrl}/api/v2/package-ids";
+            var uri = AppendAutocompletePackageIdsQueryString(serviceEndpoint, id, includePrerelease, semVerLevel);
+            var httpClient = new SimpleHttpClient();
```

Comment:
Yep! Fixed in https://github.com/NuGet/NuGet.Services.EndToEnd/pull/13/commits/dadd1cf7ee93e779c3d45266f2d6ffbe16dee691